archetypes/archetype-cdi/src/main/resources/archetype-resources/pom.xml

```diff
@@ -155,6 +155,13 @@
                 <enabled>false</enabled>
             </snapshots>
         </repository>
+        <repository>
+            <id>camel-staging</id>
```

Comment:
Let's remove this given the artifacts are now in Maven central. Same for the other archetype.

Reply:
sure - merci